mqtt/mqtt-bridge/src/client.rs

```diff
@@ -306,8 +306,8 @@ impl<H: EventHandler> MqttClient<H> {
             None
         }) {
             debug!("handling event {:?}", event);
-            if let Err(e) = self.event_handler.handle(&event).await {
-                error!(err = %e, "error processing event {:?}", event);
+            if let Err(e) = self.event_handler.handle(event).await {
+                error!(err = %e, "error processing event");
             }
```

Comment:
We aren't doing anything with the Handled level here. I'm guessing we have it for tests, and in case we need to log it in the future?

Reply:
It is logged before we consume it